server/routes/appointmentAPI.js

```diff
@@ -1,27 +1,37 @@
 const express = require("express");
 const db = require("../db/models");
 const auth = require("../middleware/auth");
-
 const ensureAppointmentExists = require("../middleware/ensureAppointmentExists");
-
 const router = express.Router();
 
 // CREATE appointment
-router.post("/api/appointment", auth, (req, res) => {
-  console.log(req.body);
+router.post("/api/appointment", async (req, res) => {
+  // req.body {eventId, name, email, notes, timezone, time}
+  try {
+    const { eventId } = req.body;
 
-  db.Appointment.create({ ...req.body })
-    .then((response) => res.send(response))
-    .catch((error) => {
-      console.log(error);
-      res.status(500).send(error);
+    //get the userId that's hosting this event
+    const { userId: hostId } = await db.EventType.findOne({ _id: eventId });

```

Comment:
You are getting the `hostId` and setting that to an object with a key of `userId ` correct?

Reply:
Just renaming. `const {userId: hostId} = await ...` is equivalent to:
```
const {userId} = await ...;
const hostId = userId;